tab.html

```diff
@@ -21,6 +21,7 @@
       <p>&#8212; <%= author %></p>
     </div>
   </script>
+  <script src="app.module.js"></script>
   <script src="background.js"></script>
```

Comment:
`background.js` only has two lines of code, let's just inline the JavaScript. Example:
```html
<script>
  var app = new App();
  app.init();
</script>
```